package.json

```diff
@@ -35,5 +35,9 @@
     "webpack": "^4.15.1",
     "webpack-cli": "^3.0.8",
     "webpack-dev-server": "^3.1.5"
+  },
+  "dependencies": {
+    "mhub": "^0.9.1",
```

Comment:
This module is a web module, it didn't need any dependencies, only Dev dependencies. Remove it and add it again with the `--dev` flag